logic/src/commonMain/kotlin/com/wire/kalium/logic/featureFlags/KaliumConfigs.kt

```diff
@@ -0,0 +1,44 @@
+package com.wire.kalium.logic.featureFlags
+
+data class KaliumConfigs(
+    val isChangeEmailEnabled: Boolean = false,
+    val isMarketingCommunicationEnabled: Boolean = false,
+    val isSSoEnabled: Boolean = false,
+    val isAccountCreationEnabled: Boolean = false,
+    val blockOnJailbreakOrRoot: Boolean = false,
+    val blockOnPasswordPolicy: Boolean = false,
+    val forceAppLock: Boolean = false,
+    val forceConstantBitrateCalls: Boolean = false,
+    val forceHideScreenContent: Boolean = false,
+    val forcePrivateKeyboard: Boolean = false,
+    val keepWebSocketOn: Boolean = false,
+    val wipeOnCookieInvalid: Boolean = false,
+    val submitCrashReports: Boolean = false,
+    val webLinkPreview: Boolean = false,
+    val developerFeaturesEnabled: Boolean = false,
+    val isLoggingEnabled: Boolean = false,
+    val isSafeLoggingEnabled: Boolean = false,
+    val enableBlacklist: Boolean = false,
+    val fileRestrictionEnabled: Boolean = false,
```

Comment:
my idea was to keep the values here so the reloaded or another client will be always reading from this data class , instead of either read it from the build config directly or from here

Reply:
> my idea was to keep the values here so the reloaded or another client will be always reading from this data class , instead of either read it from the build config directly or from here

It's better to add values with default parameters later, then adding them now and removing later leading to breaking changes.

If we remove `isMarketingCommunicationEnabled` because we find out that it's and old and unused feature that we only added because we had on Android, we will have to annotate with `@Deprecated`  first, and only remove it in a major version of Kalium.

Reply:
hmmmm true , adding will be easier than removing , so I think I will remove the things that we sure that we won't use in kalium or at all for now